includes/admin/helpers/tables/class-fees-table.php

```diff
@@ -25,16 +25,39 @@ public function __construct() {
     }
 
     public function no_items() {
+		if ( 'all' === $this->get_current_view() ) {
+            echo str_replace(
+                '<a>',
+                '<a href="' . esc_url( admin_url( 'admin.php?page=wpbdp-admin-fees&wpbdp-view=add-fee' ) ) . '">',
+                _x( 'There are no fees right now. You can <a>create one</a>, if you want.', 'fees admin', 'business-directory-plugin' )
+            );
+            return;
+        }
+
+		$current_view = $this->get_current_view();
```

Comment:
Switcher tabs are necessary UX wise. Without it its hard for a user to know what is enabled and what is disabled. There needs to be a way to know states easily for the user. When I was testing I found it hard to know what is enabled and/or disabled

Reply:
We can ask Yaro to design a background color or something on the disabled plans. I think having navigation just adds complication. We can't really compare to WooCommerce here, since products could get into the thousands. In this case, the number of plans will stay very small.

Here's a good example I found doing it this way (0:59):
https://bootstrap.brilliantdirectories.com/support/solutions/articles/12000065945-guide-4-how-to-add-membership-plans-in-admin-area